src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -97,13 +99,19 @@ public void fillFluidContainerItem(ActivateEvent event, EntityRef item, FluidCon
         if (fluidContainer.fluidType == null || fluidContainer.volume < fluidContainer.maxVolume) {
             getLiquidInReach(event.getInstigatorLocation(), event.getDirection(), 3).ifPresent(block -> {
                 EntityRef owner = item.getOwner();
+                EntityRef owner2 = owner.getOwner();
```

Comment:
`owner2` does not hold much information, and is potentially ambigious (is it another owner?). As we are interested in the fact that this should have a ClientComponent, I would call this `client`. 

Likewise, we could rename `owner` to `player` or `character`.